app/src/hid.c

```diff
@@ -66,19 +66,19 @@ int zmk_hid_unregister_mods(zmk_mod_flags modifiers)
 
 enum zmk_hid_report_changes zmk_hid_press_key(zmk_key code)
 {
-    if (code >= KC_LCTL && code <= KC_RGUI)
+    if (code >= LCTL && code <= RGUI)
     {
-        return zmk_hid_register_mod(code - KC_LCTL);
+        return zmk_hid_register_mod(code - LCTL);
     }
 
-    if (ZK_IS_CONSUMER(code))
-    {
-        LOG_DBG("Toggling a consumer key!");
-        TOGGLE_CONSUMER(code, true);
-        return Consumer;
-    }
-    else
-    {
+    // if (ZK_IS_CONSUMER(code))
```

Comment:
We'll refactor this a bit, since this won't be needed if we're not shoving a subset of consumer page values into the higher bits of the keycode constants.